client/lib/plans/plans-list.js

```diff
@@ -719,12 +719,14 @@ export const PLANS_LIST = {
 			] ),
 		getSignupFeatures: () =>
 			compact( [
+				! isEnabled( 'plans/personal-plan' ) && constants.FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY,
+				! isEnabled( 'plans/personal-plan' ) && constants.FEATURE_SPAM_AKISMET_PLUS,
 				constants.FEATURE_MALWARE_SCANNING_DAILY,
 				constants.FEATURE_AUTOMATIC_SECURITY_FIXES,
 				constants.FEATURE_VIDEO_UPLOADS_JETPACK_PRO,
 				constants.FEATURE_WORDADS_INSTANT,
 				constants.FEATURE_ADVANCED_SEO,
-				constants.FEATURE_ALL_PERSONAL_FEATURES_JETPACK,
+				constants.FEATURE_ALL_FREE_FEATURES_JETPACK,
```

Comment:
This change should also depend on `isEnabled( 'plans/personal-plan' )`, something like this:
```suggestion
				isEnabled( 'plans/personal-plan' ) ? constants.FEATURE_ALL_PERSONAL_FEATURES_JETPACK : constants.FEATURE_ALL_FREE_FEATURES_JETPACK,
```